guides/app-guides/borrow-ageur.md

```diff
@@ -0,0 +1,97 @@
+---
+description: How to borrow agEUR from the Angle App
+---
+
+# Borrowing agEUR 
+
+The [Angle Borrowing module](/borrowing-module/README.md) lets you borrow agEUR or get leverage on your crypto directly from the [Angle App](https://app.angle.money/#/borrow). 
+
+Essentially, you can deposit your crypto into what is called a vault, to get a stablecoins loan in exchange. If the value of the crypto you deposited goes below a certain amount compared to the value of your loan, you can get [liquidated](/borrowing-module/vaults/liquidations.md). 
+
+![Vaults lists](../../.gitbook/assets/vaults-list2.png)
+
+{% hint style="info" %}
+You have the possibility to use ETH and stETH instead of wETH and wstETH. In this case, transactions will require two permit or approval transactions. 
+{% endhint %}
+
+## Add collateral and borrow agEUR
+
+To borrow agEUR, you need to **deposit** collateral tokens into a **vault**. Different vaults accept different tokens which have their specific loan-to-value (LTV). This means that you are able to borrow up to a certain amount of stablecoins from the amount deposited.  
+
+For example, wETH LTV is at 84% meaning that if you deposit 1,000 € worth of wETH, you can borrow up to 840 agEUR from this vault. 
+
+You can borrow agEUR in the same transaction that you deposit collateral to your vault. Once a vault is created, it is possible to deposit more collateral without borrowing more stablecoins. 
+
+Here are the steps to follow to deposit collateral and borrow agTokens: 
+1. Select the type of vault to create, defined by the collateral and stablecoin tokens
+2. Select the `Add/Borrow` action
+3. Enter the amount of collateral you want to deposit in the first input. 
+4. Enter the amount of stablecoins you want to borrow in the second input. 
+A summary of the changes on your vault and wallet will be displayed on the right. 
+5.  Add the actions to go to the next step. You will find a summary of your transaction. You can then click on `Send` to send your transaction. 
+
+![Add/borrow agEUR](../../.gitbook/assets/add-borrow.png)
+![Confirm tx borrow agEUR](../../.gitbook/assets/confirm-tx-borrow.png)
+
+{% hint style="info" %}
+There is a minimum amount to borrow of 10,000 agEUR to limit the risk of having small bad debts that are not repaid by users nor liquidators. 
+{% endhint %}
+
+
+## Repay debt and remove collateral
+
+Once a vault is created, you can repay part of your debt up to the dust amount. You can also withdraw some collateral up to the minimum LTV. 
+
+{% hint style="info" %}
+If you want to repay all your debt and withdraw all your collateral, you can directly [close your vault](#close-vault).
+{% endhint %}
+
+When repaying your debt, you have two options: use your vault collateral, or use your debt token. If you choose to repay your debt with your vault collateral, the tokens will be swapped to debt tokens on your behalf and used to repay your debt.  
```

Comment:
Add details for what happens if you use agEUR but not enough and amount swapped + about slippage tolerance here and the fact that it uses 1Inch